spec/solver_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rspec'
+require_relative '../solver'
+
+describe Solver do
+  let(:solver) { Solver.new }
+  # context 'confirms the methods:' do
+  #   solver = Solver.new
```

Comment:
- [ ] Excellent work on including the test 👏. However, I strongly advise you to remove this commented code because leaving commented code in your code base is not a good practice and makes your code look cluttered.👌😊